packages/monaco/src/browser/monaco-outline-contribution.ts

```diff
@@ -76,37 +72,44 @@ export class MonacoOutlineContribution implements FrontendApplicationContributio
         }
     }
 
-    protected async updateOutlineForEditor(editor: EditorWidget) {
+    protected async updateOutlineForEditor(editor: EditorWidget | undefined) {
         if (this.outlineViewManager.open) {
             const model = await this.getModel(editor);
```

Comment:
we should check `editor === undefined` here, then getModel doesn't need to deal with that.